sys/net/netutils/util.c

```diff
@@ -42,8 +77,7 @@ int netutils_get_ipv6(ipv6_addr_t *addr, netif_t **netif, const char *hostname)
     }
 
 #ifdef MODULE_SOCK_DNS
-    /* hostname is not an IPv6 address */
-    if (strchr(hostname, ':') == NULL) {
+    if (strchr(hostname, ':') == NULL) { /* hostname is not an IPv6 address */
```

Comment:
Why can't this stay where it was?

Reply:
Force of habit, sorry^^ I tend to put the description for an expression next to it. I've put it back.